src/components/cc-email-list/cc-email-list.stories.js

```diff
@@ -0,0 +1,424 @@
+import './cc-email-list.js';
+import './cc-email-list.smart.js';
+import { makeStory, storyWait } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+
+const SAMPLE_EMAIL_ADDRESS = 'sample.email@clever-cloud.com';
+const ANOTHER_SAMPLE_EMAIL_ADDRESS = 'another.sample.email@clever-cloud.com';
+const YET_ANOTHER_SAMPLE_EMAIL_ADDRESS = 'yet.another.sample.email@clever-cloud.com';
+const HUGE_EMAIL_ADDRESS = `hugeemaila${'d'.repeat(500)}ress@clever-cloud.com`;
+
+const primaryAddress = { state: 'idle', address: SAMPLE_EMAIL_ADDRESS, verified: true };
+const primaryUnverified = { state: 'idle', address: SAMPLE_EMAIL_ADDRESS, verified: false };
+const secondaryAddresses = [
+  { state: 'idle', address: ANOTHER_SAMPLE_EMAIL_ADDRESS, verified: true },
+  { state: 'idle', address: YET_ANOTHER_SAMPLE_EMAIL_ADDRESS, verified: true },
+];
+const secondaryEmpty = [];
+
+export default {
+  title: '🛠 Email List/<cc-email-list>',
+  component: 'cc-email-list',
+};
+
+const conf = {
+  component: 'cc-email-list',
+};
+export const defaultStory = makeStory(conf, {
+  items: [
+    {
+      emails: {
+        state: 'loaded',
+        value: {
+          primaryAddress,
+          secondaryAddresses,
+        },
+      },
+    },
+  ],
+});
+
+export const skeleton = makeStory(conf, {
+  items: [
+    {},
+  ],
+});
+
+export const errorWithLoading = makeStory(conf, {
+  items: [
+    {
+      emails: {
+        state: 'error',
+      },
+    },
+  ],
+});
+
+export const errorWithEmptyEmail = makeStory(conf, {
+  items: [
+    {
+      emails: {
+        state: 'loaded',
+        value: {
+          primaryAddress,
+          secondaryAddresses,
+        },
+      },
+      addEmailForm: {
+        state: 'idle',
+        address: {
+          value: '',
+          error: 'empty',
+        },
+      },
+    },
+  ],
+});
+
+export const errorWithInvalidEmail = makeStory(conf, {
+  items: [
+    {
+      emails: {
+        state: 'loaded',
+        value: {
+          primaryAddress,
+          secondaryAddresses,
+        },
+      },
+      addEmailForm: {
+        state: 'idle',
+        address: {
+          value: 'invalid e-mail !!',
+          error: 'invalid',
+        },
+      },
+    },
+  ],
+});
+
+export const dataLoadedWithUnverifiedPrimaryEmailAddress = makeStory(conf, {
+  items: [
+    {
+      emails: {
+        state: 'loaded',
+        value: {
+          primaryAddress: primaryUnverified,
+          secondaryAddresses: [],
+        },
+      },
+    },
+  ],
+});
+
+export const dataLoadedWithHugeEmail = makeStory(conf, {
+  items: [
+    {
+      emails: {
+        state: 'loaded',
+        value: {
+          primaryAddress: { state: 'idle', address: HUGE_EMAIL_ADDRESS, verified: true },
+          secondaryAddresses: [
+            { state: 'idle', address: HUGE_EMAIL_ADDRESS, verified: true },
+            { state: 'idle', address: HUGE_EMAIL_ADDRESS, verified: true },
```

Comment:
If I understand correctly, `verified` property on secondary addresses are not needed... am I right? :thinking: 

(here and in other stories below)

Reply:
You're right. It is an anticipation of the future.
We have only one `EmailAddress` interface for the primary address and the secondary addresses. I did not want to create two different interfaces (one with a `verified` property and another one without).
In order to respect this interface I have to specify a `verified` property even if not used.
I'll let this like that and very soon, the property will be used once we have the right API.

Reply:
Nice, thanks for the explanation! :+1: